go/vt/sqlparser/ast.go

```diff
@@ -1504,6 +1516,31 @@ func (node *FuncExpr) IsAggregate() bool {
 	return Aggregates[node.Name.Lowered()]
 }
 
+// GroupConcatExpr represents a call to GROUP_CONCAT
+type GroupConcatExpr struct {
+	Distinct  string
+	Exprs     SelectExprs
+	OrderBy   OrderBy
+	Separator string
+}
+
+// Format formats the node
+func (node *GroupConcatExpr) Format(buf *TrackedBuffer) {
+	buf.Myprintf("group_concat(%s %v %v separator %s)", node.Distinct, node.Exprs, node.OrderBy, node.Separator)
```

Comment:
I think most of these fields are optional in the grammar: https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html#function_group-concat